shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/container/compose/DockerComposedContainer.java

```diff
@@ -54,7 +55,10 @@ public DataSource getProxyDataSource(final String databaseName) {
         HikariDataSource result = new HikariDataSource();
         result.setDriverClassName(DataSourceEnvironment.getDriverClassName(getDatabaseContainer().getDatabaseType()));
         String jdbcUrl = DataSourceEnvironment.getURL(getDatabaseContainer().getDatabaseType(), getProxyContainer().getHost(), getProxyContainer().getFirstMappedPort(), databaseName);
-        result.setJdbcUrl(StringUtils.appendIfMissing(jdbcUrl, "&rewriteBatchedStatements=true"));
+        if (DatabaseTypeUtil.isMySQL(getDatabaseContainer().getDatabaseType())) {
+            jdbcUrl = StringUtils.appendIfMissing(jdbcUrl, "&rewriteBatchedStatements=true");
+        }
```

Comment:
Could appending new jdbc url property encapsulated in databas dialect container? So there's only one code snippet in module.